website/Contribution/oca_module_lifecycle_maintainer_role.rst

```diff
@@ -0,0 +1,173 @@
+       operations (eg merging pull requests) that have impact on other
+       addons than the one they maintain.
+
+-  Maintainers may proceed to some maintenance operation with relaxed
+       code review rules (eg performing simple, low risk, forward and
+       backports, maintaining documentation, applying cosmetic
+       improvements to the code such as linting). Maintainers are
+       expected to follow the highest quality standard when performing
+       such operations.
+
+-  If there is more than one declared maintainer, they must ensure good
+       coordination between them (for instance, ensure no task are left
+       unattended on the assumption that another maintainer will do it).
+
+-  PSC members should not merge PR concerning modules with declared
+       maintainers, unless the maintainer is unreactive.
+
+-  Addons with no declared maintainers are managed by the PSC.
+
+Adding, removing maintainers
+============================
+
+-  The maintainers of an addon are declared in the **maintainers** key
+       of the manifest. This key is a list of github logins. The list of
+       declared maintainers is expected to be short (one or two in
+       general).
+
+-  New candidate maintainers can propose themselves by adding their
+       github login in the manifest, with a PR, to be approved by the
+       PSC.
+
+-  Maintainers who do not wish to continue maintaining a module must
+       remove themselves from the list through a PR, and announce it on
+       the contributors mailing list, calling for volunteers.
+
+-  PSC may remove maintainers who do not follow OCA rules or fail to
+       their maintenance duties. Such action must be taken in
+       coordination with the board.
+
+-  Current maintainers are displayed (using their avatar) in the addon
+       README.rst.
+
+Document information
+====================
+References
+----------
+|RELATED|
+
+Categories
+----------
+|CATEGORIES|
+
+Revision History
+----------------
+===== ============ ============================= ===================
+Rev.  Date         Description                   Author
+===== ============ ============================= ===================
+1.1   2018/08/24   Conversion to rst             Eric Caudal
+1.0   2018         Initial Release               Daniel Reis
+===== ============ ============================= ===================
+
+License
+-------
+This work is licensed by the
+`Odoo Community Association <https://odoo-community.org>`_
+under `Creative Commons Attribution 4.0 International License <http://creativecommons.org/licenses/by/4.0/>`_.
+
+Maintainer
+----------
+OCA, or the `Odoo Community Association <https://odoo-community.org>`_,
+is a nonprofit organization whose mission is to support the collaborative
+development of Odoo features and promote its widespread use.
+
+If you want to contribute, please check our
+`website <https://odoo-community.org/page/Contribute>`_.
```

Comment:
For the sake of simplicity, could we consider the git repo provides this metadata (rev history, license, etc), and therefore remove it from the text?

Reply:
@sbidoul We could be but we should consider that there are multiple output for this documentation: website, pdf etc. which in the end might be unlinked with the original document.
Each document might have different path.

For this reason I am not sure whether this is a good idea (not against: just not sure)

Reply:
About publishing, what's the plan to publish these documents?
1. link to the github preview from the main OCA website
2. generate a standalone html page from each document
3. generate a sphinx website with it's own menu

1 and 2 will create issues to create/maintain links between documents.

3 has the drawback of being kind of a second website.

Thoughts?

Reply:
@elicoidal @OCA/board I can setup travis but before I start something, what are your thoughts on my above question.

Reply:
@sbidoul I think the idea is right away 1 and later 2 with static pages on the current website.
for 1 and 2 indeed we will have to maintain hardcoded links between documents when necessary but IMO not many of them and most of these documents will not evolve much/often.
3 IMO is painful in terms of communication.

On my side I have not received a clear answer on the footer. 
cc @OCA/board